test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -3100,50 +3096,10 @@ static public void testIdentityObjectOnRef() throws Throwable {
 		assertTrue(refClass.isIdentity());
 	}
 
-	@Test(priority=1)
-	static public void testIsPrimitiveClassOnRef() throws Throwable {
-		String fields[] = {"longField:J"};
-		Class refClass = ValueTypeGenerator.generateRefClass("testIsPrimitiveClassOnRef", fields);
-		assertFalse(PrimitiveClass.isPrimitiveClass(refClass));
```

Comment:
Can `PrimitiveClass.isPrimitiveClass(refClass)` be replaced with `refClass.isValue()` ? 
The same comment applies to the` PrimitiveClass.isPrimitiveClass()` cases removed below.

Reply:
That scenario is covered by testIdentityObjectOnRef.

Reply:
Do we have test cases for `isValue()` and `isIdentity()` on interface, value array and ref arrays ?

Reply:
Do we have test cases for isValue() and isIdentity() on interface, value array and ref arrays ?

Reply:
Looks like there is not. I can modify these tests to use isValue instead.

Reply:
I added these tests.